### DISCUSSION TITLE
getting groups from userRepresentation

### DISCUSSION THREAD

**Discussion by tmazzz** [keycloak/keycloak#14044] [Q&A]
Im using Keycloak admin-client v12.0.4 in my Kotlin app.

I can set up a new UserRepresentation and its groups. I can add it to Keycloak and i see it there.

i can query all my users using `keycloak.realm(realm).users().list()` - this works

But whenever I try to get groups from a single UserRepresentation using `user.groups` or `user.getGroups()` i just get null response. Is this possible to see the groups from UserRepresentation?

I know I can query GroupRepresentations aswell.

**Comment by denisgmarques** [comment]
I am struggling with the same problem, any solution?

**Comment by danielFesenmeyer** [comment]
Seems to be a duplicate: https://github.com/keycloak/keycloak/discussions/23674
